packages/mask/src/extension/popups/pages/Friends/ContactCard/index.tsx

```diff
@@ -135,7 +147,7 @@ export const ContactCard = memo<ContactCardProps>(function ContactCard({
                                 underline="none"
                                 target="_blank"
                                 rel="noopener noreferrer"
-                                href={urlcat('https://web3.bio/', { s: nextId })}
+                                href={`https://web3.bio/${nextId}`}
```

Comment:
Please don't change this line, web3.bio still needs to use the search page results first because the homepage may not be queried.